src/System.Net.Http/CFNetworkHandler.cs

```diff
@@ -381,6 +384,9 @@ void HandleHasBytesAvailableEvent (object sender, CFStream.StreamEventArgs e)
 
 		void AddCookie (string value, Uri uri, string header)
 		{
+#if !DOTNET_TODO
+			// DOTNET_TODO: CookieCollection.CookieCutter is internal to mscorlib:
+			// https://github.com/microsoft/referencesource/blob/a7bd3242bd7732dec4aebb21fbc0f6de61c2545e/System/net/System/Net/cookiecontainer.cs#L632
```

Comment:
Added an issue so that I will fix it: https://github.com/xamarin/xamarin-macios/issues/8072